plugins/source/salesforce/client/client.go

```diff
@@ -16,8 +16,6 @@ import (
 	"github.com/rs/zerolog"
 )
 
-const ApiVersion = "v56.0"
```

Comment:
This change doesn't belong to this PR.
The API Version behavior change should be done in a separate PR, clearly explaining the need for this change as well as addressing the issue of the differences between the API responses for different versions.